libs/cri-api-service/src/main/java/uk/gov/di/ipv/core/library/criapiservice/dto/CredentialRequestBodyDto.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.di.ipv.core.library.criapiservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class CredentialRequestBodyDto {
```

Comment:
Wondering if we should name this more specifically something like `DcmawAsyncCredentialRequestBodyDto` for now to clearly distinguish it from existing synchronous VC requests?

Reply:
Maybe, it seemed odd to give it that specific a name in a common library, but it would be clearer. The synchronous version of the request doesn't have a body at all.

Reply:
Maybe `AsyncCredentialRequestBodyDto` rather than tying it to DCMAW?